tests/operators/test_slice_projection_op.py

```diff
@@ -0,0 +1,138 @@
+"""Tests for projection operator."""
+
+# Copyright 2024 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+import torch
+from mrpro.data import SpatialDimension
+from mrpro.operators import SliceProjectionOp
+from mrpro.operators._SliceProjectionOp import SliceGaussian
+from mrpro.operators._SliceProjectionOp import SliceInterpolate
+from mrpro.operators._SliceProjectionOp import SliceSmoothedRect
+from mrpro.utils import Rotation
+
+from tests import RandomGenerator
+from tests.helper import dotproduct_adjointness_test
+
+
+def test_slice_projection_op_basic():
```

Comment:
I don't understand this test.

Reply:
The padding was not obvious to me. Maybe add a comment. For this specific case I would have expected a slice with x = 30 and y = 20 with only ones because I am only summing up along z. I do understand that you are doing this padding to ensure it also works for arbitrary angles.

Reply:
yes, this was in the docstring at some point... but somehow it vanished while refactoring :D